src/stim/simulators/frame_simulator_pybind_test.py

```diff
@@ -216,6 +216,98 @@ def test_set_pauli_flip():
         stim.PauliString('XZ___'),
     ]
 
+def test_apply_pauli_errors():
+    sim = stim.FlipSimulator(
```

Comment:
Check corner and invalid cases. Try `pauli='I'` and `pauli=0` and `pauli="whoops"`